app/src/androidTest/java/sdp/moneyrun/MenuActivityTest.java

```diff
@@ -132,6 +125,57 @@ public void joinGamePopupIsDisplayed() {
         }
     }
 
+    @Test
+    public void filterWorks(){
```

Comment:
I think you should add one more test hat tests a case where the filter does not work

Reply:
done